.travis.yml

```diff
@@ -9,6 +14,7 @@ before_install:
     - make flake
 
 install:
+    - npm config set python /usr/bin/python2.7
```

Comment:
because some dependency (``node-gyp``) work only with Python 2.x., JS's tests and Python's tests are separeted from each other